test/e2e/spec/eid_service/main_spec.js

```diff
@@ -0,0 +1,39 @@
+describe("eID-Service: The general flow", function() {
```

Comment:
This suit just contains one positive test case. Please also add some negative checks to increase the test coverage to the cases where we need to reply faulty requests in the eid service / eid server.